api/app/routes/ocr.py

```diff
@@ -21,4 +21,4 @@ async def perform_ocr(file: UploadFile = File(...)):
     out = predictor([img])
 
     return [OCROut(box=(*word.geometry[0], *word.geometry[1]), value=word.value)
-            for word in out.pages[0].blocks[0].lines[0].words]
+            for index in range(len(out.pages[0].blocks)) for word in out.pages[0].blocks[index].lines[0].words]
```

Comment:
This piece of code is still only checking the first page only and only the first line of each block. The route is single-page only for now, so let's fix this :) I would suggest:
```python
for block in out.pages[0].blocks for line in block.lines for word in line.words
```

Reply:
Would you mind updating the PR please? 🙏

Reply:
Yes but this route is single page for now!
So this means that pages only have 1 element 👍

Reply:
Hello @SamSamhuns @assistechsol :wave: 

What do you think about my previous comment?
Let's find a solution to move forward with this PR :+1:

Reply:
Sounds good, probably a good idea to mention somewhere its only for one page tho

Reply:
Yes! Mind updating the PR then? :pray: